test/integration/targets/win_app_control/test_manifest.yml

```diff
@@ -0,0 +1,154 @@
+# These tests verify various failure conditions that will invalidate a signed manifest
+
+- name: get hash of collection module
+  ansible.builtin.stat:
+    path: '{{ local_tmp_dir }}/ansible_collections/ns/invalid_manifest/plugins/modules/module.ps1'
+    get_checksum: true
+    checksum_algorithm: sha256
+  delegate_to: localhost
+  register: module_hash_raw
+
+- name: set module hash var
+  ansible.builtin.set_fact:
+    module_hash: '{{ module_hash_raw.stat.checksum | upper }}'
+
+- name: create manifest with untrusted signature
+  ansible.builtin.import_tasks: create_manifest.yml
+  vars:
+    manifest_file: manifest_v1_ok.psd1
+    cert_name: wdac-untrusted
+
+- name: run module with untrusted signed manifest
+  ns.invalid_manifest.module:
+    input: café
+  register: res
+  failed_when:
+  - res.failed == False
+  - >-
+    res.msg != "failure during exec_wrapper: Failed to process signed manifest 'ansible_collections.ns.invalid_manifest.meta.powershell_signatures.psd1': script is not signed or not trusted to run."
```

Comment:
Just a general suggestion for resilience (and esp since we've been changing these a lot lately :laughing: ): use `res.msg is contains "blah"` or `is search "pattern1.*pattern2" to look for the important bits so that minor message formatting changes don't break everything.

Reply:
Will do, sorry muscle memory here.